frontend/src/components/chat/ChatMessage.tsx

```diff
@@ -59,12 +59,27 @@ function ChatMessage({
     }
   };
 
+  const formatTimestamp = (timestamp: string) => {
+    const date = new Date(timestamp);
+    return new Intl.DateTimeFormat(undefined, {
+      dateStyle: "short",
+      timeStyle: "short",
+    }).format(date);
+  };
+
   return (
     <div
       data-testid="message"
       className={className}
       onMouseEnter={() => setIsHovering(true)}
       onMouseLeave={() => setIsHovering(false)}
+      role="article"
```

Comment:
Why is it an article 🤔?

Reply:
Err... it was kindly suggested to me 😂

Reply:
I suppose it could be for a message bubble. You can replace the `div` with `article` instead, too

Reply:
You mean the `data-testid`? Could I leave it as is or then all the corresponding tests would need changing, too? 🤔

Reply:
I've updated the test-id's to be consistent now.

Reply:
Sorry, I didn't receive the notifications for some reason 😭.

> You mean the data-testid

No, I meant the element itself. When you set the `role` prop of an element, you are adding semantic meaning to the element. For example, assistive technologies will understand that this `div` element should be interpreted as an `article`.

What I meant is that you could just go ahead and replace the element altogether. E.g.,

```html
<article
  data-testid="message"
  <!-- ... other props ... -->
/>
```

Sorry for confusing you!

Reply:
No worries! 🙏  I'll replace "div" with "article" then.